scripts/voice/deepstack-voice-listener.sh

```
@@ -0,0 +1,586 @@
+}
+
+# ── Graceful Shutdown ──────────────────────────────────────────────
+
+RUNNING=true
+
+shutdown_handler() {
+    log INFO "Shutdown signal received"
+    RUNNING=false
+    release_lock
+    # Clean up temp files
+    rm -rf "$VOICE_TEMP"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)